server/libs/infra/src/db/config/database.config.ts

```diff
@@ -1,18 +1,27 @@
 import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
 import { DataSource } from 'typeorm';
 
-export const databaseConfig: PostgresConnectionOptions = {
+let additionalSSLDatabaseConfig;
+let baseDatabaseConfig: PostgresConnectionOptions = {
   type: 'postgres',
+  entities: [__dirname + '/../**/*.entity.{js,ts}'],
+  synchronize: false,
+  migrations: [__dirname + '/../migrations/*.{js,ts}'],
+  migrationsRun: true,
+};
+
+let envBasedDatabaseConfig = {
   host: process.env.DB_HOSTNAME || 'immich_postgres',
   port: parseInt(process.env.DB_PORT || '5432'),
   username: process.env.DB_USERNAME,
   password: process.env.DB_PASSWORD,
   database: process.env.DB_DATABASE_NAME,
-  entities: [__dirname + '/../**/*.entity.{js,ts}'],
-  synchronize: false,
-  migrations: [__dirname + '/../migrations/*.{js,ts}'],
-  migrationsRun: true,
   connectTimeoutMS: 10000, // 10 seconds
```

Comment:
Can you move this up into the baseConfig?

Reply:
oops. Missed that from rebase.